package.json

```diff
@@ -49,9 +49,10 @@
     "source-map-support": "0.5.21"
   },
   "devDependencies": {
-    "@emotion/react": "11.10.5",
-    "@mui/icons-material": "5.11.0",
-    "@mui/material": "5.11.2",
+    "@emotion/react": "11.10.6",
+    "@emotion/styled": "11.10.6",
```

Comment:
📝 `@emotion/styled` is peer dependencies of MUI.

https://github.com/mui/material-ui/blob/d0c2ee1e772fc1e2b043fe274ee269f0c169782a/packages/mui-material/package.json#L58-L60